lib/ready.js

```diff
@@ -80,6 +80,9 @@ class Ready extends EventEmitter {
     opt.name = name || cacheKey;
     const timer = setTimeout(() => this.emit('ready_timeout', opt.name), opt.timeout);
     const cb = once(err => {
+      if (err != null && !(err instanceof Error)) {
+        err = new Error(err);
```

Comment:
Once err is undefined, this will be broken.

Should be `err !== null && err !== undefined` instead.